commands/commandWithTrace.class.js

```diff
@@ -0,0 +1,27 @@
+import { Command } from 'commander';
+import logger from "../logger.js";
+
+class CommandWithTrace extends Command {
```

Comment:
I believe that functionality should be optional, so we can create 2 base commands, one is simply BaseCommand and the other one CommandWithTrace that extends BaseCommand

Reply:
and follow your naming convention everywhere in the code